extensions/ssh-tunnels/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# SSH Tunnels Changelog
+
+## [08/28/22] Initial Release
```

Comment:
Wrong format, should be: `## [Initial Release] - 2022-09-01`

I'm not able to edit this PR since you locked it 🙂

Reply:
Oh no! I'm not sure how I locked it but I'll get that fixed.